content/streamlit-cloud/get-started/manage-your-app.md

```diff
@@ -176,3 +177,82 @@ To unfavorite an app, click the star (⭐) again.
 Click [here](/knowledge-base/deploy/upgrade-streamlit-version-on-streamlit-cloud) to learn more about upgrading the Streamlit version of your app on Streamlit Cloud.
 
 </Tip>
+
+## Analytics Modal
+
+Once you start or are invited to a Streamlit workspace, you have access to 2 types of analytics:
```

Comment:
Feels like this could be said easier as "Once you have access to a Streamlit workspace..."

Unless I'm misunderstanding how the permission model works

Reply:
It did originally say "Once you have access to a Streamlit workspace". I'll find out if there's a reason the phrasing was edited and base any changes off that